Makefile

```diff
@@ -194,6 +194,9 @@ upload-datadog-secrets: check-env-vars ## Decrypt and upload Datadog credentials
 	$(if $(wildcard ${DATADOG_PASSWORD_STORE_DIR}),,$(error Password store ${DATADOG_PASSWORD_STORE_DIR} does not exist))
 	@scripts/upload-datadog-secrets.sh
 
+upload-tracker-token: check-env-vars ## Decrypt and upload Pivotal tracker API token to S3
+	pass pivotal/tracker_token | aws s3 cp - "s3://${DEPLOY_ENV}-state/tracker_token"
+
```

Comment:
* Lack of .PHONY: .PHONY us useful for the case you might have a file in the same directory with the same name as the task. For `upload-tracker-token` it's highly unlikely so I assume we can avoid it.
* Same file approach: we usually create a separate script when the command is more complex. For a one-liner, it didn't seem necessary.
* Using the pass assuming it's the one: I used the same approach as in [environment.sh](https://github.com/alphagov/paas-cf/blob/4d5f8dad330b6bcae94c977c246c2f71e1453047/concourse/scripts/environment.sh#L40) where `$HOME/.paas-pass` is assumed by default